jest_config/__fixtures__/tokenMigrationMultiTx.ts

```diff
@@ -0,0 +1,50 @@
+import { REPV1UUID } from '@config';
+import { repTokenMigrationConfig } from '@features/RepTokenMigration/config';
+import { createApproveTx, createRepMigrationTx } from '@features/RepTokenMigration/helpers';
+import { fApproveERC20TxResponse, fREPTokenMigrationTxResponse } from '@fixtures';
+import { ITokenMigrationFormFull, ITxStatus, ITxType, TxParcel } from '@types';
+import { generateUUID, inputGasLimitToHex, inputNonceToHex } from '@utils';
+
+
+import { fAccount } from './account';
+import { fAssets } from './assets';
+import { fNetwork } from './network';
+
+export const fTokenMigrationTxs = (): TxParcel[] => {
```

Comment:
Hmm, will these not be generated by `useTxMulti` on test? Is it necesary to hard code them?

Reply:
`useTxMulti` doesn't actually create txs, the hook just handles input txs to then process / order / sign / broadcast / watch tx hashes / proceed to next step. The txs are actually created in the stepper config - so it is necessary to create them before testing them.